webapp/src/components/jira_ticket_tooltip/jira_ticket_tooltip.tsx

```diff
@@ -151,155 +144,107 @@ export default class TicketPopover extends React.PureComponent<Props, State> {
             return null;
         }
 
-        let totalString = 0;
-        let totalHide = 0;
-        const labelList = labels.map((label: string, key: number): JSX.Element => {
-            if (totalString < 3) {
-                totalString++;
-                return (
-                    <span
-                        key={key}
-                        className='popover-labels__label-list'
-                    >
-                        {label}
-                    </span>);
-            }
-
-            totalHide++;
-            if (key === labels.length - 1) {
-                const moreLabels = `+${totalHide}more`;
-                return (
-                    <span
-                        key={key}
-                        className='popover-labels__label-list'
-                    >
-                        {moreLabels}
-                    </span>);
-            }
+        return (
+            <div className='popover-labels__label'>
+                {
+                    labels.map((label: string, key: number): ReactNode => {
+                        if (key < 3 || (key === labels.length - 1 && labels.length - 3 > 0)) {
```

Comment:
```suggestion
                        if (key < 3 || (key === labels.length - 1 && labels.length - 3)) {
```

Reply:
Negative values are not considered as falsy in JS.